id-repository/bioextractor-service/Dockerfile

```diff
@@ -44,12 +57,30 @@ ENV extraction_format_env=${extraction_format}
 
 ENV server_port_env=${server_port}
 
-ENV bio_sdk_folder_env=${bio_sdk_folder}
+ARG boisdk_local_dir=biosdk-client
 
-ENV work_dir_env=/
+ENV biosdk_local_dir_name=${boisdk_local_dir}
+
+#ENV bio_sdk_folder_env=${bio_sdk_folder}
+ENV biosdk_zip_file_path=${biosdk_zip_path}
+
+#ENV work_dir_env=/home/${container_user}
 
 ENV current_module_env=bioextractor-service
 
+# install packages and create user
+RUN apt-get -y update \
+&& apt-get install -y unzip \
+&& groupadd -g ${container_user_gid} ${container_user_group} \
+&& useradd -u ${container_user_uid} -g ${container_user_group} -s /bin/sh -m ${container_user} \
+&& adduser ${container_user} sudo \
+&& echo "%sudo ALL=(ALL) NOPASSWD:/home/${container_user}/${boisdk_local_dir}/install.sh" >> /etc/sudoers

```

Comment:
@mandeepdhiman123 please update the variable name boisdk_local_dir to biosdk_local_dir